diff --git a/composer.json b/composer.json
index 72695d1..5bedc02 100644
--- a/composer.json
+++ b/composer.json
@@ -47,7 +47,8 @@
     "watch": "php core/console --watch",
     "start": "php core/console --server  --quiet & php core/console --watch",
     "post-install-cmd": [
-      "PatternLab\\Installer::postInstallCmd"
+      "PatternLab\\Installer::postInstallCmd",
+      "composer run generate"
     ],
     "post-update-cmd": [
       "PatternLab\\Installer::postUpdateCmd"
diff --git a/gulpfile.js b/gulpfile.js
index 192452e..9618920 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -47,11 +47,6 @@ const errorHandler = error => {
   this.emit("end");
 };
 
-// Use the Pattern Lab PHP command to generate the pattern library
-const plPhp = () => {
-  return exec("php core/console --generate");
-};
-
 const copyUswdsFonts = () => {
   return gulp.src(`${uswds}/fonts/**/*`).pipe(gulp.dest(config.fonts.public_fonts));
 };
@@ -175,7 +170,7 @@ const serve = cb => {
   });
 };
 
-const build = gulp.series(plPhp, copyUswdsFonts, copyUswdsImages, copyIconSprite, plCss, plJs);
+const build = gulp.series(copyUswdsFonts, copyUswdsImages, copyIconSprite, plCss, plJs);
 
 exports.build = build;
 exports.default = gulp.series(build, serve, watch);
diff --git a/source/_twig-components/functions/create_attribute.function.php b/source/_twig-components/functions/create_attribute.function.php
index 721472c..1d4be7c 100644
--- a/source/_twig-components/functions/create_attribute.function.php
+++ b/source/_twig-components/functions/create_attribute.function.php
@@ -6,4 +6,4 @@
 use Drupal\Core\Template\Attribute;
 $function = new Twig_SimpleFunction('create_attribute', function ($attributes = []) {
   return is_array($attributes) ? new Attribute($attributes) : $attributes;
-});
\ No newline at end of file
+});
